### PR TITLE
Bump decidim to 0.28.2

### PR DESCRIPTION
**Pull request by HHassig**
Other than the one time actions for the production server, i ran into the following issue:
`rails aborted!
NameError: uninitialized constant Decidim::OfficialImageHeaderUploader

    validates_upload :official_img_header, uploader: Decidim::OfficialImageHeaderUploader
                                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Did you mean?  Decidim::OfficialImageFooterUploader
/platoniq/decidim-recharge/app/models/decidim/organization.rb:50:in `<class:Organization>'
/platoniq/decidim-recharge/app/models/decidim/organization.rb:7:in `<module:Decidim>'
/platoniq/decidim-recharge/app/models/decidim/organization.rb:3:in `<main>'
/platoniq/code/platoniq/decidim-recharge/config/environment.rb:7:in `<main>'
Tasks: TOP => assets:precompile => shakapacker:compile => environment
(See full trace by running task with --trace)
`

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -13,17 +13,17 @@ gem "decidim-conferences", DECIDIM_VERSION
...
 # gem "decidim-templates", DECIDIM_VERSION
 
-gem "decidim-alternative_landing", git: "https://github.com/Platoniq/decidim-module-alternative_landing", branch: "main"
+gem "decidim-alternative_landing", git: "https://github.com/Platoniq/decidim-module-alternative_landing"
 gem "decidim-decidim_awesome", git: "https://github.com/decidim-ice/decidim-module-decidim_awesome", branch: "main"
-gem "decidim-direct_verifications", git: "https://github.com/Platoniq/decidim-verifications-direct_verifications", branch: DECIDIM_BRANCH
-gem "decidim-extra_user_fields", git: "https://github.com/Platoniq/decidim-module-extra_user_fields", branch: DECIDIM_BRANCH
-gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer", branch: "master"
+# gem "decidim-direct_verifications", git: "https://github.com/Platoniq/decidim-verifications-direct_verifications"
+gem "decidim-extra_user_fields", git: "https://github.com/PopulateTools/decidim-module-extra_user_fields", branch: DECIDIM_BRANCH
```
Comment (fblupi) [inline Gemfile:19]: You should use our fork because we include some of our custom fields. We need to upgrade it to the 0.28 branch too

File: Gemfile [hunk 2]
```diff
@@ -13,17 +13,17 @@ gem "decidim-conferences", DECIDIM_VERSION
 # gem "decidim-initiatives", DECIDIM_VERSION
 # gem "decidim-templates", DECIDIM_VERSION
 
-gem "decidim-alternative_landing", git: "https://github.com/Platoniq/decidim-module-alternative_landing", branch: "main"
+gem "decidim-alternative_landing", git: "https://github.com/Platoniq/decidim-module-alternative_landing"
 gem "decidim-decidim_awesome", git: "https://github.com/decidim-ice/decidim-module-decidim_awesome", branch: "main"
-gem "decidim-direct_verifications", git: "https://github.com/Platoniq/decidim-verifications-direct_verifications", branch: DECIDIM_BRANCH
-gem "decidim-extra_user_fields", git: "https://github.com/Platoniq/decidim-module-extra_user_fields", branch: DECIDIM_BRANCH
-gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer", branch: "master"
+# gem "decidim-direct_verifications", git: "https://github.com/Platoniq/decidim-verifications-direct_verifications"
```
Comment (fblupi) [inline Gemfile:18]: Why did you remove this one?

File: app/models/decidim/organization.rb
Comment (fblupi) [inline app/models/decidim/organization.rb:1]: This monkey-patch was introduced here: https://github.com/Platoniq/decidim-recharge/commit/3c296ee359cc5e191a8b5da3a6837af8f115e8c7

It wasn't included to the overrides spec and we are paying the consequences (a great lesson to learn).

You should check the differences between the file when it was included and the one decidim had on that version. And then adapt it with the changes of the 0.28 on that file. It may solve your problem in the description of the PR.

**Review by fblupi — CHANGES_REQUESTED** [review]
The pipeline is failing too. You should solve it too.

Did you run the tests locally? I'm almost 100% some of the tests will fail (overrides spec for example).

File: Gemfile (hunk 2 above)
Comment (HHassig) [reply Gemfile:18]: It appears this gem isn't ready for 0.28

Comment (fblupi) [reply Gemfile:18]: Then, we need to upgrade it

**Comment by HHassig** [comment]
I think the tests are failing at `bundle install` because i have had to roll back rubocop versions in the Gemfile.lock, similar to decidim-ub's gemfile issue when we updated decidim-awesome:
[https://github.com/Platoniq/decidim-ub/pull/13/commits/1e08c197269f42c53a7673d1038a77971a553dcc](url)
